test/lib/Helpers.scala

```diff
@@ -2,39 +2,35 @@ package lib
 
 import java.net.URL
 
-import com.google.common.io.Files
-import com.madgag.git._
 import com.squareup.okhttp.OkHttpClient
 import lib.Implicits._
 import lib.gitgithub.GitHubCredentials
-import org.eclipse.jgit.api.Git
 import org.eclipse.jgit.lib.{AbbreviatedObjectId, ObjectId}
-import org.eclipse.jgit.transport.RemoteRefUpdate
 import org.kohsuke.github.GHIssueState.OPEN
 import org.kohsuke.github._
-import org.scalatest.concurrent.{Eventually, IntegrationPatience, ScalaFutures}
-import org.scalatest.{BeforeAndAfterAll, Inspectors}
+import org.scalatest.Inspectors
+import org.scalatest.concurrent.{Eventually, ScalaFutures}
+import org.scalatest.time.{Millis, Seconds, Span}
 import org.scalatestplus.play._
 
 import scala.collection.convert.wrapAll._
 import scala.concurrent.Future
 
 case class PRText(title: String, desc: String)
 
-trait Helpers extends PlaySpec with OneAppPerSuite with Inspectors with ScalaFutures with Eventually with IntegrationPatience with BeforeAndAfterAll {
+trait Helpers extends PlaySpec with OneAppPerSuite with Inspectors with ScalaFutures with Eventually {
 
-  val testRepoNamePrefix = "prout-test-"
+  implicit override val patienceConfig =
+    PatienceConfig(timeout = scaled(Span(12, Seconds)), interval = scaled(Span(850, Millis)))
```

Comment:
Not sure why I switched to custom `PatienceConfig` here, rather than use the fairly-slow-paced stuff in [`IntegrationPatience`](https://github.com/scalatest/scalatest/blob/d5f38074ea9057a0f56045f1bfd1f6eb2e24a782/jvm/core/src/main/scala/org/scalatest/concurrent/IntegrationPatience.scala#L72-L75), which would currently give us:

```scala
    PatienceConfig(
      timeout = scaled(Span(15, Seconds)),
      interval = scaled(Span(150, Millis))
    )
```

...the customised patience conf we've got here times out a bit more quickly, but retries more slowly. I think the retires need to be even slower, given that we're encountering GitHub's secondary rate limits:

* https://docs.github.com/en/rest/guides/best-practices-for-integrators?apiVersion=2022-11-28#dealing-with-secondary-rate-limits
* https://docs.github.com/en/rest/overview/resources-in-the-rest-api?apiVersion=2022-11-28#secondary-rate-limits